km-specs-tests/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+  kotlin("jvm")
+}
+
+val compileTestKotlin: KotlinCompile by tasks
+compileTestKotlin.kotlinOptions {
+  freeCompilerArgs = listOf("-XXLanguage:+InlineClasses", "-Xjvm-default=enable")
+  jvmTarget = "1.8"
+}
+
+dependencies {
+  testImplementation(deps.kotlin.stdlib)
+  testImplementation(project(":kotlinpoet"))
+  testImplementation(project(":kotlinpoet-km"))
+  testImplementation(project(":kotlinpoet-km-specs"))
+  testImplementation(project(":kotlinpoet-elementhandler-elements"))
+  testImplementation(project(":kotlinpoet-elementhandler-reflective"))
+  testImplementation(deps.kotlin.junit)
+  testImplementation(deps.test.truth)
+  testImplementation(deps.test.compileTesting)
+}
+
+repositories {
```

Comment:
Root `build.gradle` applies a `repositories` block to all subproject, so I believe re-applying it here is not necessary.

Reply:
c2c64bd